packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -544,6 +555,11 @@ export interface ProjectProps extends CommonProjectProps {
    * @see https://docs.aws.amazon.com/codebuild/latest/userguide/sample-multi-in-out.html
    */
   secondaryArtifacts?: BuildArtifacts[];
+
+  /**
+   * The VPC to which this project should be attached.
+   */
+  vpcConfig?: VPCConfig
```

Comment:
We usually prefer to have two properties “vpc” and “vpcPlacement” instead of this. Take a look at lambda and ecs as examples.

Reply:
+1 to what Elad said. See [here](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda/lib/function.ts#L133).

Reply:
One more thing: this should be added to `CommonProjectProps`, not `ProjectProps`.